src/crimson/osd/osd_operations/client_request.h

```diff
@@ -52,6 +52,102 @@ class ClientRequest final : public PhasedOperationT<ClientRequest>,
     friend class HistoricBackend;
   };
 
+  /**
+   * instance_handle_t
+   *
+   * Client request is, at present, the only Operation which can be requeued.
+   * This is, mostly, fine.  However, reusing the PipelineHandle or
+   * BlockingEvent structures before proving that the prior instance can create
```

Comment:
Sounds like there is a missing part in _proving that the prior instance (...) can create_.